src/test/resources/messages.properties

```diff
@@ -41,4 +39,14 @@ too.big.email=Email length should be less than 129 characters. Please enter vali
 
 valid.email.regexp=^[\\w-\\+]+(\\.[\\w]+)*@[\\w-]+(\\.[\\w]+)*(\\.[a-zA-Z]{2,})$
 valid.password.regexp=(?=.*?[0-9])(?=.*?[a-z])(?=.*?[A-Z])(?=\\S+$).+.{8,30}
-valid.username.regexp=(?=\\S+$).{2,50}
\ No newline at end of file
+valid.username.regexp=(?=\\S+$).{2,50}
+
+## Exceptions messages
```

Comment:
Apply here my comments from the same file but for prod.

Reply:
Performed